packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -367,6 +367,23 @@ abstract class SliverMultiBoxAdaptorWidget extends RenderObjectWidget {
   }
 }
 
+/// A base class for sliver that have multiple box children.
+///
+/// Helps subclasses build their children lazily using a [SliverChildDelegate].
+abstract class SliverMultiBoxAdaptorWidget extends SliverMultiKeepAliveBoxAdaptorWidget {
```

Comment:
it's weird that the class with the less specific name is inheriting from the one with the more specific name